server_admin/topics/authentication/flows.adoc

```diff
@@ -179,14 +172,16 @@ image:images/Passwordless-browser-login.png[]
 
 After entering the username, the way this flow works is the following:
 
-* If the user has any WebAuthn credentials recorded, he will be able to use any of them to log in directly. This is the password-less login.
-  The user also has a drop-down allowing him to select "Password with OTP". He can do this because the "WebAuthn" execution and the "Password with OTP"
+* If the user has any WebAuthn passwordless credentials recorded, he will be able to use any of them to log in directly. This is the password-less login.
+  The user also has a way allowing him to select "Password with OTP". He can do this because the "WebAuthn Passwordless" execution and the "Password with OTP"
```

Comment:
Change " The user also has a way allowing him to select"  to "The user can instead select"
Change "He" to "The user"

Reply:
Fixed